Documentation/Source/Support/UDim4.md

```diff
@@ -22,4 +22,43 @@ The Y coordinate of the UDim4.
 ## Methods
 
 ### [Vector2](http://wiki.roblox.com/index.php?title=API:Vector2) ToVector2([GuiObject](http://wiki.roblox.com/index.php?title=API:Class/GuiObject) guiObject)
-Returns the absolute value (in pixels) of the UDim4, relative to the parent of `guiObject`. If `guiObject` is parented to `nil`, a zeroed Vector2 (`0, 0`) is returned.
\ No newline at end of file
+Returns the absolute value (in pixels) of the UDim4, relative to the parent of `guiObject`. If `guiObject` is parented to `nil`, a zeroed Vector2 (`0, 0`) is returned.
+
+# UDim4Enforcer
+The UDim4Enforcer class is designed to assist in applying a [UDim4](udim4.md) value to a GuiObject's [Position](http://wiki.roblox.com/index.php?title=API:Class/GuiObject/Position) and [Size](http://wiki.roblox.com/index.php?title=API:Class/GuiObject/Size) properties. It will automatically apply UDim4s to the object and keep them up to date until it is destroyed.
+
+## Constructors
+### UDim4:MakeEnforcer([GuiObject](http://wiki.roblox.com/index.php?title=API:Class/GuiObject) object, UDim4 position, UDim4 size)
+Creates a new UDim4Enforcer for `object` with position `position` and size `size`. If either `position` or `size` are `nil`, the corresponding properties of the GuiObject will not be changed when the enforcer acts.
+
+## Properties
+### [GuiObject](http://wiki.roblox.com/index.php?title=API:Class/GuiObject) Object *[readonly]*
```

Comment:
The readonly status is not a tag; prefix the declaration:
`readonly [GuiObject](http://wiki.roblox.com/index.php?title=API:Class/GuiObject) Object`